docs/root/start/simple_example.rst

```diff
@@ -0,0 +1,259 @@
+
+  The ``name`` key names the virtual host for statistics reporting.
+
+  The ``domains`` key contains a list of domains (host/authority header) that will be matched to this virtual
+  host.
+
+  The ``routes`` key contains a list of :ref:`Routes <envoy_api_msg_route.Route>` that are matched, in order, for incoming requests. The first
+  route that matches is used.
+
+  The ``match`` key provides the :ref:`RouteMatch parameters <envoy_api_msg_route.RouteMatch>` used to select the route.
+
+  The ``route`` key provides the :ref:`RouteAction parameters <envoy_api_msg_route.RouteAction>` invoked when the route is selected.
+
+  The ``http_filters`` key contains a list of
+  :ref:`HttpFilter <envoy_api_msg_config.filter.network.http_connection_manager.v2.HttpFilter>` for the connection manager.
+  In this case
+  the built-in :ref:`envoy.router <config_http_filters_router>` filter is specified which
+  implements HTTP forwarding by following the instructions specified above in the route table.
+
+
+.. code-block:: yaml
+
+      clusters:
+      - name: service_google
+        connect_timeout: 0.25s
+        type: LOGICAL_DNS
+        # Comment out the following line to test on v6 networks
+        dns_lookup_family: V4_ONLY
+        lb_policy: ROUND_ROBIN
+        hosts: [{ socket_address: { address: google.com, port_value: 443 }}]
+        tls_context: { sni: www.google.com }
+
+..
+
+  The ``clusters`` key specifies a list of :ref:`clusters <arch_overview_cluster_manager>`. You can refer to the
+  :ref:`cluster configuration <envoy_api_file_envoy/api/v2/cds.proto>` details.
+
+  The ``name`` key is required and must be unique across all clusters. It is used when emitting statistics.
+
+  The ``connect_timeout`` key specifies a timeout value for new network connections to hosts in the cluster.
+
+  The ``type`` key specifies the :ref:`service discovery type <arch_overview_service_discovery_types>`
+  to use for resolving the cluster.
+
+  The ``dns_lookup_family`` key specifies the :ref:`DnsLookupFamily <envoy_api_enum_Cluster.DnsLookupFamily>`
+  which is the DNS IP address resolution
+  policy.
+
+  The ``lb_policy`` key specifies the :ref:`load balancer type <arch_overview_load_balancing_types>`
+  to use when picking a host in the cluster.
+
+  The ``hosts`` key specifies the :ref:`host address <envoy_api_msg_core.Address>`.
+  If the service discovery type is
+  :ref:`STATIC<envoy_api_enum_value_Cluster.DiscoveryType.STATIC>`,
+  :ref:`STRICT_DNS<envoy_api_enum_value_Cluster.DiscoveryType.STRICT_DNS>`
+  or :ref:`LOGICAL_DNS<envoy_api_enum_value_Cluster.DiscoveryType.LOGICAL_DNS>`,
+  then ``hosts`` is required.
+
+  The ``tls_context`` key specifies the :ref:`TLS configuration <envoy_api_msg_auth.UpstreamTlsContext>`.
+  for connections to the upstream cluster. If no TLS
+  configuration is specified, TLS will not be used for new connections.
+
+
+Using the Envoy Docker Image
+----------------------------
```

Comment:
Given the context above, I wonder if we should rename this to "Using your own configuration file with the Envoy Docker image" because that's really what this does? Something like that ...

Reply:
I moved this section out to Getting Started and changed it a little as it's generally useful for any example. I did follow your suggestion about the title.